pkgs/servers/bazel-remote/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage {
```

Comment:
Please use the new [`buildGoModule`](https://kalbas.it/2019/03/17/announcing-the-new-golang-infrastructure-buildgomodule/) instead of buildGoPackage.

Reply:
@kalbasit Can that information be included in https://nixos.org/nixpkgs/manual/#sec-language-go?

Reply:
@uri-canva it's already there, it's just not on the manual online as it's still version 18.09. I published the manual [here](https://kalbas.it/nixpkgs/pull/57080/doc/#sec-language-go), it's also described on the announcement on my [blog](https://kalbas.it/2019/03/17/announcing-the-new-golang-infrastructure-buildgomodule/).

Reply:
Perfect, thank you!